Codetour for E2E Pipline (#212)
Azure/ALZ-Bicep: docs/wiki/CodeTour.md

<!-- markdownlint-disable -->
## Contents
<!-- markdownlint-restore -->

- [Contents](#contents)

> - [E2E Testing pipeline tour](#e2e-testing-pipeline-tour)

Reviewer: ```suggestion
```

> ---

Reviewer: ```suggestion
```

> As par of the ongoing development of ALZ Bicep, we are beginning to add [CodeTour](https://marketplace.visualstudio.com/items?itemName=vsls-contrib.codetour) tours of the code. Tours can be found in [tours_repo]. 

Reviewer: ```suggestion
As part of the ongoing development of ALZ-Bicep, we are beginning to add [CodeTour](https://marketplace.visualstudio.com/items?itemName=vsls-contrib.codetour) tours of the code, where appropriate. Tours can be found in [tours_repo]. 
```

> 

Reviewer: Can we add a link and a small blurb about how to use a codetour?

Like https://marketplace.visualstudio.com/items?itemName=vsls-contrib.codetour&ssr=false#user-content-opening-tours ?

> ## E2E Testing pipeline tour

Reviewer: ```suggestion
## E2E Testing Pipeline Tour
```
